core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -204,6 +204,10 @@ public Slave getNode() {
         }
     }
 
+    public TaskListener getListener() {
```

Comment:
/**
 * Return the {@code TaskListener} for this SlaveComputer.  This is generally a {@link StreamTaskListener} that wraps {@link #super.getLog()}
 * @return the {@code TaskListener} object, never null.
 * @since // TODO populate when merging (or 2.8 if you feel lucky)
*/
public....